shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -220,16 +222,7 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
         }
 
         if (object.children != null) {
-            List<SemanticsObject> childrenInTraversalOrder =
-                new ArrayList<SemanticsObject>(object.children);
-            Collections.sort(childrenInTraversalOrder, new Comparator<SemanticsObject>() {
-                public int compare(SemanticsObject a, SemanticsObject b) {
-                    final int top = Integer.compare(a.globalRect.top, b.globalRect.top);
-                    // TODO(goderbauer): sort right-to-left in rtl environments.
```

Comment:
Should we keep this TODO since the algorithm still has a LTR bias for the equal case?

Reply:
Don't we defer to tree order for the equal case?

Reply:
Yes, we will.  I need to move this geometric comparison from here to the framework side for the geometric comparison to happen within the LTR/RTL context, as we discussed, and then if they are geometrically equal, or there is no Directionality in force, then we will defer to tree order.